Makefile.am

```diff
@@ -156,6 +156,11 @@ BATS = \
 	test/functional/checkupdate/chk-update-client-certificate.bats \
 	test/functional/completion/completion-basic.bats \
 	test/functional/hashdump/hashdump-file-hash.bats \
+	test/functional/install/install.bats \
+	test/functional/install/install-bad.bats \
+	test/functional/install/install-multiple.bats \
+	test/functional/install/install-no-fullfile-fallbacks.bats \
+	test/functional/install/install-no-zero-packs.bats \
```

Comment:
don't you think that it would be better to add these tests to the verify directory instead of install? I see we already have a couple of tests related to install in the verify directory:
https://github.com/clearlinux/swupd-client/blob/master/Makefile.am#L211